plugins/optimization-detective/docs/introduction.md

```diff
@@ -186,7 +186,7 @@ add_filter( 'query_vars', function ( $query_vars ) {
 
 Then in your template code you may continue to use `isset( $_GET['thank_you'] )` but you might as well use `get_query_var( 'thank_you' )` instead.
 
-When an `od_url_metrics` post has not been updated in a month then it is garbage-collected, since it is likely the original URL has gone away.
+When an `od_url_metrics` post has not been updated within the default value i.e., a month or the filtered value via `od_url_metric_garbage_collection_ttl` then it is garbage-collected, since it is likely the original URL has gone away.
```

Comment:
```suggestion
When an `od_url_metrics` post has not been modified for three (3) months, it will then be garbage-collected since it is likely the original URL for the metrics has gone away. If you have a site that gets very little traffic and you want to increase the TTL for garbage collection, you can use the `od_url_metric_garbage_collection_ttl` filter to increase this.
```